packages/hint-manifest-icons/src/_locales/en/messages.json

```diff
@@ -11,6 +11,14 @@
         "description": "Report message when an icon request return an status code !== 200",
         "message": "Icon could not be fetched (status code: $1)."
     },
+    "iconPurposeDuplicate": {
+        "description": "Report message when an icon has a duplicate value in the property purpose",
+        "message": "Duplicate value(s) '$1' found in icon purpose property."
```

Comment:
The convention is to use `$1` at the end so localization is easier.
@johnemau do you have any suggestions?

Reply:
what about `Duplicate value(s) found in icon purpose porperty: $1`?

Reply:
SGTM